### ISSUE TITLE
Need to customize Link popup

### ISSUE THREAD

**Issue by umarjavedse** [sstur/react-rte#392]
Using RichTextEditor to convert content to markdown and vice versa. `Open Link in new tab` and `No Follow` are working with markdow. So, I want to remove these both checkboxes from `Link` popup.
![image](https://user-images.githubusercontent.com/23642660/100099976-ef935080-2e81-11eb-9a28-6d8e23f55d8c.png)

Please let me know how can I manage that.
Thanks in advance.

**Comment by umarjavedse** [comment]
@sstur can you or any other collaborator please share your thought on this issue.

**Comment by umarjavedse** [comment]
Created PR #395  for this issue

**Comment by Hammad3912** [comment]
Yes am also stuck on this issue since 1 week please approve or resolve this as soon as possible Thanks

**Comment by umarjavedse** [comment]
I used this package two month back and log this issue. It's seems like support is not more looking into issues. PR is also pending, kindly look into this. Waiting for positive response. Thanks.

**Comment by phronaeus** [comment]
Hello Umar, did you ever end up resolving this issue? Since the PR hasn't been merged, how did you work around the problem?

**Comment by umarjavedse** [comment]
@phronaeus No this issue is still there.

**Comment by iMinosGFX** [comment]
I found a way a little complicated to remove the links, here is my trick: 

Give a class to the toolbar 
`toolbarClassName="editorToolbar"`

Then in scss (can also be done in css) : 
.editorToolbar{
  ### ### div[class*="ButtonGroup"]{
      &:nth-child(3){
        div[class*="InputPopover__root"]{
          div[class*="InputPopover__checkOption"]{
            display: none;
          }
        }
      }
    }
}


Modify the nth-child(3) according to the index where your **ButtonGroup** link is in the toolbar (depending on what you display or not in it, it can change)

![Screenshot_4](https://user-images.githubusercontent.com/35743637/117849951-1fde7800-b285-11eb-8fce-b82c28b38cfd.png)
